hedyorg/hedy
# 🧹 Remove superflous code in generated pygame code frontend

Fixes (part of) 3966

**How to test**
Follow these steps to verify this PR works as intended:
* Run `if a is pressed print 'a' else print 'other'` in Level 5 / Introduction
* Press `a` on keyboard.

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)


If you're unsure about any of these, don't hesitate to ask. We're here to help!
